lambdas/post_to_slack/post_to_slack.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+
+"""
+
+import json
+import pprint
+import os
```

Comment:
should be an extra newline between `import os` and `from botocore.vendored import requests`